### PR TITLE
[it] Readme.md Added words

### PR DESCRIPTION
**Pull request by codeadmin-peritiae**

### CODE REVIEW

**Comment by gmb-ftcont** [comment]
lgtm

File: site/it/README.md [hunk 1]
```diff
@@ -45,19 +57,25 @@ Alcune parole fanno parte di un vocabolario tecnico inglese e pertanto non hanno
...
+*   namespace(s)
 *   notebook(s)
-*   overfit(ting)
+*   open source
 *   pattern(s)
 *   pixel(s)
-*   pull request(s)
-*   repository(ies)
+*   plug-in(s)
+*   previewer(s)
```
Comment (bhack) [inline site/it/README.md:66]: Anteprima?

Comment (codeadmin-peritiae) [reply site/it/README.md:66]: Anteprima is preview: the noun.
Previewer (something or somebody who previews) has not a correspondence in Italian...

**Comment by codeadmin-peritiae** [comment]
Good morning @lamberta, 
the CI/labeler chesk failed on this file.
I just added some words as usual.
Can you explain me why did it fail? (To avoid similar mistakes in the future).
Thank you.
G.

**Pull request merged**